src/arch/xtensa/include/arch/init.h

```diff
@@ -37,8 +37,7 @@ static inline void exception(void)
 
 	/* now panic and rewind 8 stack frames. */
 	/* TODO: we could invoke a GDB stub here */
-	panic_rewind(SOF_IPC_PANIC_EXCEPTION, 8 * sizeof(uint32_t),
```

Comment:
@abonislawski shouldn't we also take care of the comment saying rewing 8 stack frames?

Reply:
sorry, fixed